pom.xml

```diff
@@ -796,9 +796,9 @@
                 <!-- Non-standard plugins -->
 
                 <plugin>
-                    <groupId>com.jayway.maven.plugins.android.generation2</groupId>
+                    <groupId>com.simpligility.maven.plugins</groupId>
```

Comment:
you are right, we don't need it here, since we only have it as a dependency in the android examples now. I will remove it.